src/sentry/auth/idpmigration.py

```diff
@@ -6,11 +6,11 @@
 
 from sentry import options
 from sentry.models import Organization, OrganizationMember, User
-from sentry.utils import redis
+from sentry.utils import json, redis
 from sentry.utils.email import MessageBuilder
 from sentry.utils.http import absolute_uri
 
-_REDIS_KEY = "verificationKeyStorage"
+REDIS_KEY = "verificationKeyStorage"
```

Comment:
did you intend to make this public?

Reply:
i think i made the change for it to be public when i imported it in auth_helper, seeing its no longer used there, ill revert it back to being private